R/brm.R

```diff
@@ -231,6 +231,11 @@
 #'   (e.g., initial values, number of iterations, control arguments, ...). A
 #'   known limitation is that a refit will be triggered if within-chain
 #'   parallelization is switched on/off.
+#' @param file_auto Logical If \code{TRUE} file parameter will be calculated
+#'   automatically with the parameters given omitting the ones that do not change
```

Comment:
the doc here is unclear. can you try to improve it? If you are unsure how, I can edit it directly